metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -36,7 +36,10 @@ def something(since):
 
 
 def get_optional_collectors():
-    return os.environ.get('METRICS_UTILITY_OPTIONAL_COLLECTORS', 'main_jobevent').split(',')
+    opt_collector = os.environ.get('METRICS_UTILITY_OPTIONAL_COLLECTORS', 'main_jobevent')
+    if not opt_collector or opt_collector.isspace():
+        opt_collector = 'main_jobevent'
+    return opt_collector.split(',')
```

Comment:
With `METRICS_UTILITY_OPTIONAL_COLLECTORS=""` this should return an empty array, not `['main_jobevent']`.
(The default should only kick in with `unset METRICS_UTILITY_OPTIONAL_COLLECTORS`.)

(ie it should still be possible to disable the default optional collector, without adding any others)